openxla_patches/cuda_driver.diff

```diff
@@ -0,0 +1,22 @@
+diff --git a/xla/stream_executor/cuda/cuda_driver.cc b/xla/stream_executor/cuda/cuda_driver.cc
+index e60e17d64..0ebe57e07 100644
+--- a/xla/stream_executor/cuda/cuda_driver.cc
++++ b/xla/stream_executor/cuda/cuda_driver.cc
+@@ -902,7 +902,7 @@ static CUmemLocationType ToCudaLocationType(
+       return CU_MEM_LOCATION_TYPE_INVALID;
+     case GpuDriver::MemLocationType::kDevice:
+       return CU_MEM_LOCATION_TYPE_DEVICE;
+-#if CUDA_VERSION >= 12000
++#if CUDA_VERSION >= 12030
```

Comment:
what's this change for?

Reply:
@vanbasten23 can you take a look at this one? If we do need to change this, it is better to submit the change internally and remove this patch later.

Reply:
thanks, submitted change internally and will remove this patch in the OpenXLA-pin update after Dec8